### DISCUSSION TITLE
卡顿、音画不同步问题

### DISCUSSION THREAD

**Discussion by ChaXxl** [the1812/Bilibili-Evolved#4211] [综合讨论 / General]
* **最初的使用情况：** 每次打开一个视频总会卡一下（就是先有声音，视频卡几秒钟，之后才会同步）
* **再后来使用久了之后：** 直接音画不同步了，就很难受，而且有时候视频还不会缓冲。是有时不同步、有时又同步
>  我 Windows、MacOS、Chrome、Edge，都试过了，全都会在使用时间久了之后变音画不同步，应该和系统、浏览器没啥关系
>  但是当我把 Bilibili-Evolved 给禁用之后就音画同步了，也不会卡

兄弟们来看看到底是啥问题，音画不同步太难受了，但是让我放弃使用 Bilibili-Evolved 那是不可能的！
![图层 2](https://github.com/the1812/Bilibili-Evolved/assets/40326898/67269f6c-97f4-4edb-af20-e4ecde424a07)

**Comment by the1812** [comment]
我猜是 b 站分配的 CDN 问题, 时不时会有那种 1MB 内容要加载 20 多秒的情况, 然后声音会继续放但画面停住

**Reply by the1812** [reply]
遇到了一次, b 站给的视频流有问题, 调试看了视频在正常解码播放, 但是画面一直是同一帧. 更换清晰度使用不同的视频流倒是能解决

**Comment by MaxHasBeenUsed** [comment]
我经常遇到这个问题，解决方法是按一下左方向键。

**Reply by Alexius66** [reply]
你可以尝试使用[【哔哩哔哩】屏蔽视频PCDN地址](https://greasyfork.org/zh-CN/scripts/500519-%E5%93%94%E5%93%A9%E5%93%94%E5%93%A9-%E5%B1%8F%E8%94%BD%E8%A7%86%E9%A2%91pcdn%E5%9C%B0%E5%9D%80)以切换为官方CDN来解决这一问题
